zerver/actions/realm_settings.py

```diff
@@ -403,6 +403,9 @@ def do_change_realm_org_type(
         extra_data={"old_value": old_value, "new_value": org_type},
     )
 
+    event = dict(type="realm", op="update", property="org_type", value=org_type)
+    transaction.on_commit(lambda: send_event(realm, event, active_user_ids(realm.id)))
+
```

Comment:
I kept this decorated as `transaction_atomic(durable=True)` after reading through some [recent conversations in CZO](https://chat.zulip.org/#narrow/stream/3-backend/topic/Choosing.20the.20approach.20for.20atomic.28.29.20in.20do_.2E.2E.2E.2E.20functions/near/1312009), but I thought it was worth noting here to make sure that was the right call.

Reply:
That sounds correct to me.